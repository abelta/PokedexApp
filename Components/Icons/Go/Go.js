import React from 'react';
import { Image } from 'react-native';
import styles from './styles';

// eslint-disable-next-line global-require
const Go = () => <Image source={require('./right-arrow.png')} style={styles.go} />;

export default Go;
