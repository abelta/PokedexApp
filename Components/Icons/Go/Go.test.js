import React from 'react';
import renderer from 'react-test-renderer';
import Go from '.';

describe('Go', () => {
  const tree = renderer.create(<Go />).toJSON();
  it('renders correctly', () => {
    expect(tree).toMatchSnapshot();
  });
});
