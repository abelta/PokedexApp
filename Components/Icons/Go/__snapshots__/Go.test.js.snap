// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Go renders correctly 1`] = `
<Image
  source={1}
  style={
    Object {
      "height": 20,
      "width": 20,
    }
  }
/>
`;
